build/cron-gen.module.js

```diff
@@ -705,7 +706,87 @@ var CronGenTimeSelect = function CronGenTimeSelect($scope, cronGenService) {
 };
 CronGenTimeSelect.$inject = ["$scope", "cronGenService"];
 
-angular.module('angular-cron-gen', []).service('cronGenService', CronGenService).component('cronGenTimeSelect', {
+angular.module('angular-cron-gen', ['pascalprecht.translate']).config(["$translateProvider", function ($translateProvider) {
+    $translateProvider.translations('en', {
```

Comment:
I think that it's possible, but I must check. Probably I should find some more smart way to passing translations I will make some tries!